.github/PULL_REQUEST_TEMPLATE/frontend_template.md

```diff
@@ -1,46 +1,46 @@
 <!--- Provide a general summary of your changes in the Title above -->
 <!-- This is a template, feel free to remove unused sections or add information you think it's needed -->
 
+### Linked PRs
+<!-- (Optional) Please list any related PRs here:
+- PR #123 (Title and link)
+- PR #456 (Title and link)
+-->
+
 ## What does this PR do?
 <!--- Describe your changes in detail. Explain the process which led you to decide to do the change as you did -->
 
+<!--
+**Before and After Screenshots:**
+- **Before:** ![before](link-to-before-screenshot)
+- **After:** ![after](link-to-after-screenshot)
+-->
+
+## Why is this change necessary?
+<!-- (Optional) Describe the reasoning behind this change. Explain the motivation, the problem it addresses, or the benefits it provides. 
+This section helps reviewers understand the broader context and importance of the changes. -->
+
 ## Associated ticket number and/or AirBrake error?
 <!--- If this has a related ticket/task, add it here -->
 <!--- Also add any AirBrake errors that will be fixed by this -->
```

Comment:
More of a comment for the general template, but I don't know if everyone knows we can write `[ABC-123]` in the description so that the Jira bot picks up the ID and replaces it with a link.

Reply:
Hmm interesting. I always put the link manually. I'll try this in my next PR.